website/docs/r/kp_key.html.markdown

```diff
@@ -11,8 +11,7 @@ description: |-
 
 Create, or delete a Key Protect standard or root key. To use the `ibm_kp_key` resource, the region parameter in the `provider.tf` file must be set to the same region that your Key Protect service instance. If region parameter is not specified, `us-south` is used as default. If the region in the `provider.tf` file is different from the Key Protect instance, the instance cannot be retrieved by  Terraform and the  Terraform action fails.
 
-**Note**
-
+~>**Deprecated:**
```

Comment:
I'm guessing this is a typo
```suggestion
**Deprecated:**
```

Reply:
Not a typo. I dont know how it works, but I think there is some extra markdown formatting that gets done in the repo to get this formatting. 

At least, other examples in this repo use this
![Screenshot 2023-09-05 at 1 48 03 PM](https://github.com/IBM-Cloud/terraform-provider-ibm/assets/38149204/3210bbd0-556b-4a0f-a5cd-e17b6342f421)

Reply:
Without the ~> it will just be an ordinary bold text